src/interactivity/onecore/BgfxEngine.hpp

```diff
@@ -71,9 +71,6 @@ namespace Microsoft::Console::Render
         [[nodiscard]] HRESULT GetFontSize(_Out_ COORD* const pFontSize) noexcept override;
         [[nodiscard]] HRESULT IsGlyphWideByFont(const std::wstring_view glyph, _Out_ bool* const pResult) noexcept override;
 
-    protected:

```

Comment:
Didn't you forget to remove some of them from this file?

Reply:
Yes, I can't compile Bgfx & WDDM. I don't know if it can compile or not. So I just left it as it is.